eng/common/scripts/Save-Package-Properties.ps1

```diff
@@ -0,0 +1,40 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $serviceName,
+  [Parameter(Mandatory=$True)]
+  [string] $OutDirectory
+)
+
+. (Join-Path $PSScriptRoot common.ps1)
+$allPackageProperties = @{}
+if (Test-Path "Function:Get-AllPkgProperties")
```

Comment:
I don't think we need to test for this function as we expect it to always be defined in the common scripts.